src/seedu/addressbook/parser/Parser.java

```diff
@@ -156,7 +167,6 @@ private static boolean isPrivatePrefixPresent(String matchedPrefix) {
         return new HashSet<>(tagStrings);
     }
 
-
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.